linked-list.rb

```diff
@@ -63,12 +63,36 @@ def display
    end
 
    def include?(key)
+     current = @head
+     full_list = []
+     while current.next_node != nil
+         full_list += [current.value.to_s]
+         (return true) if current.value == key
+         current = current.next_node
+     end
+     return false
    end
```

Comment:
This looks good -- however you don't need `full_list` in this method at all. so just remove lines 67 and 69